tests/acceptance/AdminSample.feature

```diff
@@ -68,6 +68,14 @@ Feature: admin page for samples
     And I should see "Attribute name for the input animal=\tiger\ is not valid - please select a valid attribute name!"
     And I should see "Attribute name for the input plant=\rose\ is not valid - please select a valid attribute name!"
 
+  @ok
+  Scenario: display input format error when update
+    Given I am on "/adminSample/update/id/432"
+    When I fill in the field of "name" "Sample[name]" with "test"
+    And I press the button "Save"
+    And I wait "1" seconds
+    Then I should see "test"
+
```

Comment:
Thanks for adding the acceptance test, but regarding the title of this scenario, I think it is misleading as it looks to me is for testing for the invalid sample ID input.

**suggestion:** Is it better to have 2 scenarios, for example as below:
```
Scenario: display updated sample name when update
    Given I am on "/adminSample/update/id/432"
    When I fill in the field of "name" "Sample[name]" with "test"
    And I press the button "Save"
    And I wait "1" seconds
    Then I should see "test"

Scenario: display error/warning message of empty sample name when update
    Given I am on "/adminSample/update/id/432"
    When I fill in the field of "name" "Sample[name]" with ""
    And I press the button "Save"
    Then I should see "Empty sample name input is not alllowed" (something like this)
  ```

Reply:
my mistake, it was just a copy/paste and I forgot to update the name 😅

Reply:
Ken's first scenario could be improved upon by checking that you are re-directed to the sample view page after successful update of Sample ID using:
```
Then I am on "/adminSample/view/id/432"
And I should see "test"
```